index.html

```diff
@@ -4,7 +4,111 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Personal Website</title>
+
+    <link rel="stylesheet" href="assets/styles/main.css" />
+    <link rel="stylesheet" href="assets/styles/header.css" />
+    <link rel="stylesheet" href="assets/styles/footer.css" />
+    <script
+      src="https://kit.fontawesome.com/0069c24e64.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Personal Website Christian Daum</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <a class="navbar-brand" href=""> Christian Daum | WebDev</a>
+      <button class="navbar-toggler">
+        <span class="toggler-icon"></span>
+      </button>
+
+      <nav class="navbar">
+        <ul>
+          <li>
+            <a href="">Skills</a>
+          </li>
+          <li>
+            <a href="">Interests</a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+          <li>
+            <a href="">Bio</a>
+          </li>
+          <li>
+            <a href="">Travels</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- <section class="name"></section>
+    <section class="photo"></section>
+    <section class="about"></section>
+    <section class="work"></section>
+    <section class="education"></section>
+    <section class="community"></section>
+    <section class="skills"></section> -->
+
+    <section class="hero">
+      <h1>Hero</h1>
+      <h3></h3>
+    </section>
+
+    <section class="photo">
+      <h1>Pic</h1>
+      <h3></h3>
+    </section>
+
+    <section class="skills">
+      <h1>Skills</h1>
+      <h3></h3>
+    </section>
+
+    <section class="interests">
+      <h1>Interests</h1>
+      <h3></h3>
+    </section>
+
+    <section class="contact">
+      <h1>Contact</h1>
+      <h3></h3>
+    </section>
+
+    <section class="links">
+      <h1>Links</h1>
+      <h3></h3>
+    </section>
+
+    <section class="about-me">
+      <h1>About Me, Myself, and I</h1>
+      <h3></h3>
+    </section>
+
+    <section class="cv">
+      <h1></h1>
+      <h3></h3>
+      <section class="work">work</section>
+      <section class="education">education</section>
+    </section>
+
+    <section class="travels">
+      <h1>Travels</h1>
+      <h3></h3>
+
+      <section class="backpacking-nz">Backpacking New Zealand</section>
+      <section class="interail-italy">Backpacking Italy</section>
+      <section class="interail-europe">Backpacking Europe via Interail</section>
```

Comment:
Maybe this could be a list?

Reply:
Will work on a solution for that tomorrow... I wanted to jazz this up with some pics to transport a more personal touch - but that would be far too heavy for a list - this why I was 'contemplating' (sure) about the usage of subsections...